src/Discord.Net/Entities/Messages/IUserMessage.cs

```diff
@@ -0,0 +1,24 @@
+// todo: docs
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Discord
+{
+    public interface IUserMessage : IMessage
+    {
+        string JumpUrl { get; } // review: use System.Uri for this?
```

Comment:
Is there any benefit for having it as a Uri?

Reply:
We already went down that rabbit hole of swapping some of the appropriate things to `System.Uri` and general consensus was not a fan of it.

Reply:
Of note: Even the CoreFX API team doesn't like `System.Uri`.